### PR TITLE
Lint policy recommendation PySpark code with flake8

### PR DESCRIPTION
**Pull request by dreamtalen**
Lint policy recommendation PySpark code with flake8, flake8 configuration is copied from NSX openstack project at https://opendev.org/x/vmware-nsx/src/branch/master/tox.ini.

### CODE REVIEW

File: plugins/policy-recommendation/policy_recommendation_job.py [hunk 1]
```diff
@@ -97,29 +109,66 @@ def get_protocol_string(protocolIdentifier):
...
+        [flow.destinationPodNamespace, flow.destinationPodLabels]
+    )
     return (dst, (src, ""))
 
-# Combine the ingress row (src, "") with the egress row ("", dst) to a network peer row (src, dst)
+
+# Combine the ingress row (src, "") with the egress row ("", dst) to a network
+# peer row (src, dst)
+
+
```
Comment (yuntanghsu) [inline plugins/policy-recommendation/policy_recommendation_job.py:171]: extra empty lines?

Comment (dreamtalen) [reply plugins/policy-recommendation/policy_recommendation_job.py]: Good catch! Thanks

**Review by yuntanghsu — COMMENTED** [review]
LGTM

**Review by yuntanghsu — APPROVED** [review]

**Review by heanlan — APPROVED** [review]

**Pull request merged**